mod_ci/controllers.py

```diff
@@ -442,6 +442,33 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
             platform=platform, id=test.id))
 
 
+def ci_badge(status, platform, test_type, branch="master"):
+    if test_type == TestType.commit:
+        if platform is TestPlatform.linux:
+            if status == Status.SUCCESS:
+                svg = open('../static/badges/Linux_success.svg')
+            elif status == Status.FAILURE:
+                svg = open('../static/badges/Linux_failed')
+            else:
+                svg = open('../static/badges/Linux_error.svg')
+            svg.shutil(https://github.com/CCExtractor/sample-platform/static/badges,
+                        sampleplatform.ccextractor.org/buildtest/results)
+
+        else:
+            if status == Status.SUCCESS:
+                svg = open('../static/badges/Windows_success.svg')
+            elif status == Status.FAILURE:
+                svg = open('../static/badges/Windows_failed.svg')
+            else:
+                svg = open('../static/badges/Windows_error.svg')
+            svg.shutil(https://github.com/CCExtractor/sample-platform/static/badges,
```

Comment:
This makes zero sense. Please stop adding commits until you are certain that it all would work, and will yield *no* programming errors.

Reply:
okay